GOVERNANCE.md

```diff
@@ -166,19 +166,18 @@ Provide a summary of the nominee's contributions. For example:
   organization
 * Other participation in the wider Node.js community
 
-Mention @nodejs/collaborators in the issue to notify other collaborators about
-the nomination.
+If the private discussion receives some support (in form of explicit comments or
+thumbs-up reactions) and no explicit block after a few days, add a comment in the
+private discussion stating you're planning on opening a public issue, e.g.
+"I see a number of approvals and no block, I'll be opening a public
+nomination issue if I don't hear any objections in the next 72 hours".
+After sufficient time has passed, open an issue in the [nodejs/node][] repository.
+Provide the same summary of the nominee's contributions.
 
 The nomination passes if no collaborators oppose it after one week. In the case
```

Comment:
Does this still apply to public nomination, or both? If both, what's the purpose of private nomination?

Reply:
Gather objections before the public nomination is opened

Reply:
If there are no objections on private nomination, can someone object on public nomination duration?

Reply:
Yes, if someone (collaborator or not) thinks a nomination should not proceed for whatever reason, they should obviously share their concerns – the goal of this PR is not to reduce the time when an objection can be made, it's to make it more likely it will happen before the public issue is opened.

Reply:
Yes, but this doesn't disallow people from objecting in public nominations. It just legalizes private nomination part, previously it was not mandatory but recommended if nominator is interested in reducing friction. 

So my question is, does this change really help? If yes, to whom? To nominator? To nominee? To collaborators?

It seems like making it mandatory or not doesn't change anything, but add yet another step.

Reply:
I believe it does help yes, and I disagree it adds "yet another step", I don't think any nominations in the past 3 years did not start with a private discussion.

Reply:
Note that I’m saying it helps, not that it solves it.